apps/server/src/modules/toasts/toasts.controller.ts

```
@@ -37,4 +38,23 @@ import { Toast } from './entities/toast.entity';
   ): Promise<Toast> {
     return this.toastsService.adminEditToast(id, toastData);
   }
+
+  @Get('personal-record/:userId')
+  async getPersonalRecord(@Param('userId') userId: string): Promise<number> {
+    try {
+      return await this.toastsService.getPersonalRecord(userId);
+    } catch (error) {
+      throw new NotFoundException(
```

is error.message undefined?
i think you can just throw the error